### PR TITLE
Check markup and spelling on every PR

### PR DESCRIPTION
**Pull request by asvetlov**
Fix for #2339

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -143,6 +143,7 @@ jobs:
     env:
     - docs
     script:
+    - towncrier --yes
```
Comment (webknjaz) [inline .travis.yml:146]: Do we have it installed at this point?

Comment (webknjaz) [reply .travis.yml:146]: Nevermind, just checked: it's okay.

File: CHANGES/2396.doc [hunk 2]
```diff
@@ -1 +1 @@
-Improve tutorial docs, using literalinclude to link to the actual files.
+Improve tutorial docs, using `literalinclude` to link to the actual files.
```
Comment (webknjaz) [inline CHANGES/2396.doc:1]: This change is unrelated.

Comment (asvetlov) [reply CHANGES/2396.doc:1]: It's a fix for already committed CHANGES.
`literalinclude` is a spelling error without backticks.
